src/Test/NuGet.Config

```diff
@@ -4,9 +4,10 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="restsdk" value="https://pkgs.dev.azure.com/mseng/PipelineTools/_packaging/nugetvssprivate/nuget/v3/index.json" />
+    <add key="BuildXL.Selfhost" value="https://pkgs.dev.azure.com/ms/BuildXL/_packaging/BuildXL.Selfhost/nuget/v3/index.json" />
```

Comment:
Was this the special library that needed to be added?  Can you elaborate on why?

Reply:
vss-api-netcore (the API contracts for ADO) depends on something called BuildXL. BuildXL depends on RocksDbSharp. I published a new version of BuildXL because vss-api-netcore depended on it. However, that new version of BuildXL depended on a new version of RocksDbSharp that was not available in our existing set of NuGet feeds. This new feed allows us to get the newest versions of these packages.